hepcrawl/pipelines.py

```diff
@@ -13,13 +13,15 @@
 See: http://doc.scrapy.org/en/latest/topics/item-pipeline.html
 """
 
+from __future__ import absolute_import, division, print_function
+
 import datetime
 import json
 import os
 
 import requests
 
-from crawler2hep import crawler2hep
+from hepcrawl.crawler2hep import crawler2hep
```

Comment:
```python
from .crawler2hep import crawler2hep
```

^ that means that it's importing from the current package, whatever the name of it is, the other, always imports from an 'installed' package called hepcrawl.